src/digitalmarketplace/components/keyword-search/_keyword-search.scss

```diff
@@ -0,0 +1,260 @@
+$input-size: 40px;
+$large-input-size: 50px;
+
+.dm-keyword-search {
+  position: relative;
+  margin-bottom: 30px;
+}
+
+.dm-keyword-search__label {
+  @include govuk-font($size: 19, $line-height: $input-size);
+  display: block;
+  background: govuk-colour("white");
+  color: $govuk-text-colour;
+
+  h1 {
+    @include govuk-font($size: 19, $line-height: $input-size);
+    margin: 0;
+  }
+
+  .js-enabled & {
+    position: absolute;
+    left: 2px;
+    top: 2px;
+    bottom: 2px;
+    padding-left: govuk-spacing(3);
+    z-index: 1;
+    color: $govuk-secondary-text-colour;
+  }
+
+  // match label colour with the label component colour
+  // when javascript is enabled and inline_label option  is set to false
+  .js-enabled .dm-keyword-search--separate-label & {
+    color: $govuk-text-colour;
+  }
+}
+
+.dm-keyword-search__input[type="search"] { // overly specific to prevent some overrides from outside
+  @include govuk-font($size: 19, $line-height: (28 / 19));
+
+  padding: 6px;
+  margin: 0;
+  width: 100%;
+  height: $input-size;
+  border: $govuk-border-width-form-element solid $govuk-input-border-colour;
+  background: govuk-colour("white");
+  border-radius: 0; //otherwise iphones apply an automatic border radius
+  box-sizing: border-box;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+
+  // the .focus class is added by JS and ensures that the input remains above the label once clicked/filled in
+  &:focus,
+  &.focus {
+    z-index: 2;
+  }
+
+  &:focus {
+    outline: $govuk-focus-width solid $govuk-focus-colour;
+    // Ensure outline appears outside of the element
+    outline-offset: 0;
+    // Double the border by adding its width again. Use `box-shadow` for this // instead of changing `border-width`
+    // Also, `outline` cannot be utilised here as it is already used for the yellow focus state.
+    box-shadow: inset 0 0 0 $govuk-border-width-form-element;
+
+    @include govuk-if-ie8 {
+      // IE8 doesn't support `box-shadow` so double the border with
+      // `border-width`.
+      border-width: $govuk-border-width-form-element * 2;
+    }
+  }
+}
+
+.dm-keyword-search__submit {
+  border: 0;
+  cursor: pointer;
+  border-radius: 0;
+  position: relative;
+  padding: 0;
+  width: $input-size;
+  height: $input-size;
+  background-image: url("../images/search-button.png");
+  background-repeat: no-repeat;
+  background-position: 2px 50%;
+  text-indent: -5000px;
+  overflow: hidden;
+
+  @include govuk-device-pixel-ratio {
+    background-size: 52.5px auto;
+    background-position: 115% 50%;
+  }
+
+  &:focus {
+    z-index: 2;
+    outline: $govuk-focus-width solid $govuk-focus-colour;
+    // Ensure outline appears outside of the element
+    outline-offset: 0;
+    // Double the border by adding its width again. Use `box-shadow` for this // instead of changing `border-width` - this is for consistency with
```

Comment:
This comment seems to be cut off halfway through?

Reply:
Victim of a copy/paste here. I assume the consistency is between the submit button and the input - it's similarly cut off on the govuk-publishing-components file.